diff --git a/node_modules/react-native-header-search-bar/lib/src/HeaderSearchBar/HeaderSearchBar.js b/node_modules/react-native-header-search-bar/lib/src/HeaderSearchBar/HeaderSearchBar.js
index 24a45a6..8f76b2a 100644
--- a/node_modules/react-native-header-search-bar/lib/src/HeaderSearchBar/HeaderSearchBar.js
+++ b/node_modules/react-native-header-search-bar/lib/src/HeaderSearchBar/HeaderSearchBar.js
@@ -28,17 +28,17 @@ const HeaderSearchBar = props => {
     <Androw style={shadowStyle(shadowColor)}>
       <View style={container(backgroundColor)}>
         <View style={styles.containerGlue}>
-          <TouchableOpacity onPress={onPressHamburgerIcon}>
+          {/* <TouchableOpacity onPress={onPressHamburgerIcon}>
             {iconComponent || (
               <Icon
                 size={25}
                 name="menu"
                 type="Entypo"
-                color="black"
+                color="red"
                 {...props}
               />
             )}
-          </TouchableOpacity>
+          </TouchableOpacity> */}
           <View style={styles.titleContainer}>
             <Text style={firstTitleStyle(firstTitleColor, firstTitleFontSize)}>
               {firstTitle}
diff --git a/node_modules/react-native-header-search-bar/lib/src/HeaderSearchBar/HeaderSearchBar.style.js b/node_modules/react-native-header-search-bar/lib/src/HeaderSearchBar/HeaderSearchBar.style.js
index 5698c25..1a937fa 100644
--- a/node_modules/react-native-header-search-bar/lib/src/HeaderSearchBar/HeaderSearchBar.style.js
+++ b/node_modules/react-native-header-search-bar/lib/src/HeaderSearchBar/HeaderSearchBar.style.js
@@ -1,58 +1,59 @@
 import {
-  hasNotch,
-  getStatusBarHeight
+    hasNotch,
+    getStatusBarHeight
 } from "@freakycoder/react-native-helpers";
 
 export const shadowStyle = shadowColor => {
-  return {
-    shadowColor,
-    shadowRadius: 9,
-    shadowOpacity: 0.2,
-    shadowOffset: {
-      width: 0,
-      height: 4
-    }
-  };
+    return {
+        // paddingBottom: 8,
+        // shadowColor,
+        // shadowRadius: 8,
+        // shadowOpacity: 0.1,
+        // shadowOffset: {
+        //     width: 0,
+        //     height: 0
+        // }
+    };
 };
 
 export const container = backgroundColor => {
-  return {
-    height: 210,
-    width: "100%",
-    backgroundColor,
-    borderBottomEndRadius: 24,
-    borderBottomStartRadius: 24
-  };
+    return {
+        height: 190,
+        width: "100%",
+        backgroundColor,
+        borderBottomEndRadius: 24,
+        borderBottomStartRadius: 24,
+    };
 };
 
 export const firstTitleStyle = (color, fontSize) => {
-  return {
-    color,
-    fontSize,
-    fontWeight: "bold"
-  };
+    return {
+        color,
+        fontSize,
+        fontWeight: "bold"
+    };
 };
 
 export const secondTitleStyle = (color, fontSize) => {
-  return {
-    fontSize: 18,
-    marginTop: 3,
-    color: "#34343b",
-    fontWeight: "bold"
-  };
+    return {
+        fontSize: 18,
+        marginTop: 3,
+        color: "#34343b",
+        fontWeight: "bold"
+    };
 };
 
 export default {
-  containerGlue: {
-    marginLeft: 24,
-    marginTop: hasNotch() ? getStatusBarHeight() : 0
-  },
-  titleContainer: {
-    marginTop: 12,
-    marginLeft: 3
-  },
-  searchboxContainer: {
-    top: 8,
-    margin: 16
-  }
+    containerGlue: {
+        marginLeft: 24,
+        marginTop: hasNotch() ? getStatusBarHeight() : 0,
+    },
+    titleContainer: {
+        marginTop: 24,
+        marginLeft: 3,
+    },
+    searchboxContainer: {
+        top: 0,
+        margin: 16
+    }
 };
